### PR TITLE
Ternary expression can be used to determine which component should be used

### PR DESCRIPTION
**Pull request by Simek**
Check: http://jsfiddle.net/Simek/e0nragnp/1/

### CODE REVIEW

File: docs/tips/03-if-else-in-JSX.md [hunk 1]
```diff
@@ -33,23 +33,23 @@ That's not valid JS. You probably want to make use of a ternary expression:
 React.render(<div id={condition ? 'msg' : ''}>Hello World!</div>, mountNode);
 ```
 
-If a ternary expression isn't robust enough, you can use `if` statements to determine which
-components should be used.
+You can also use ternary expression to determine which tags or components should be used.
```
Comment (zpao) [inline docs/tips/03-if-else-in-JSX.md]: A few things.

1. They're all components
2. I'd really like to keep an `if/else` case in here just so people can see this pattern. Ternaries get super ugly beyond the simple, self-closing case. So let's talk about both in the text here, and have both examples.
3. I don't like your formatting in the code block so I'll suggest something else (though I think this should stay an if/else and leave the other one as a ternary.
4. cc @chenglou who might have some other thoughts on this page


```js
var loginInformation = isLoggedIn ?
  <cite>Hello!</cite> :
  <strong>Please log in</strong>;
```

Comment (irvinebroque) [inline docs/tips/03-if-else-in-JSX.md]: +1 to keeping `if/else`. I point people coming from traditional templating systems to this page all the time, and `if/else` makes JSX feel more familiar.

If there is a ternary example, it makes sense to keep it within the return statement:

		return (
		  <nav>
		    {isLoggedIn ? <cite>Hello!</cite> : <strong>Log in, please...</strong>}
		    {isLoggedIn ? null : <SignUpButton/>;}
		  </nav>
		);

Comment (chenglou) [inline docs/tips/03-if-else-in-JSX.md]: Agree with @zpao. Not sure about the new ternary below. I feel like an `if` is fine.

**Comment by zpao** [comment]
@Simek, are you interested in addressing the comments?

**Comment by zpao** [comment]
No updates and don't feel like this is important enough to push through, closing

**Pull request closed without merging**